src/utils/mtev_time.c

```diff
@@ -228,14 +228,23 @@ fetch_cclock_scale(int cpuid, struct cclock_scale *cs) {
 }
 #endif
 
-#if defined(linux) || defined(__linux) || defined(__linux__)
+#if defined(BSD) || defined(__FreeBSD__)
```

Comment:
Note that MacOS also defines `BSD`, so maybe this needs to go after the `__MACH__` case?